decentraland/unity-renderer
# feat: multiple prefabs support for channel entries

## What does this PR change?

The channel lists (conversation & browser) now supports a configurable scriptable object that provides different types of entry instances depending on the channel id.

## How to test the changes?

1. Open the conversation list
2. All channels should be displayed correctly
3. The nearby channel prefix should be `~`
4. The nearby channel should not have the option to leave the channel
5. Go to channel browser (search button)
6. Search for channels, everything should work correctly

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
